cmake-modules/AddAzureSDK.cmake

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# SPDX-License-Identifier: MIT
+#
+# How to use: Copy this file to a cmake modules folder within your project.
+# - Add `list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}/your-cmake-modules-folder")`
+# - Call this module by doing `include(AddAzureSDK)` from root CMakeLists.txt
+# - Link your application against your cmake targets like `target_link_libraries(yourApp PRIVATE az_core)`
+#
+# This will create a _deps folder for the generated project and Azure SDK will be checked out in there.
+# When calling `cmake --build .`, Azure SDK will be built before your application and become available.
+#
+# Cmake options for Azure SDK for C can be set together with your application cmake options.
+#
+# Note: Update GIT_TAG to the expected version from Azure SDK for C
+#
+
+include(FetchContent)
+FetchContent_Declare(cppstoragesdk
+    GIT_REPOSITORY      https://github.com/Azure/azure-sdk-for-cpp.git
+    GIT_TAG             azure-template_1.0.0-beta.3)
```

Comment:
cpp?

Let me know once this is ready for review.

Reply:
copy past issue.  Fixed it now.

Reply:
If you prefer, turn such PRs to draft to convey it is WIP (work in progress), and once it is ready, flip it back.

Reply:
it was not a draft. It was ready before, but with a bug on it!  I missed to update the module because I finished out of the project where I was using it.  Then I just update the documentation about it without updating the code.  My bad